### PR TITLE
Work branch for creating HTML structure and CSS style

### PR DESCRIPTION
**Pull request by fedgut**
Please review this 5th project of the Microverse CSS and HTML course.

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hi there @fedgut, nice job on this project 👍 💯 
But before merging to master, we need to make some changes on it to make it better.

- [x] You must use the `article` tag for each article in the page.

- [x]  There are some errors in your CSS code. Check them using this CSS [validator](https://jigsaw.w3.org/css-validator/) . Ignore the warnings.

**[FOR FUTURE PROJECTS]**
Nice job not naming the branch of dev/developer. But next time try to give a feature name for the branch(e.g. 'created_sidebar' or 'added_comments_section'), just like it said in the Github flow [instructions](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/)

Please make these changes and submit for code review again. Happy coding. 💻

File: index.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Heatmap</title>
+</head>
+<body>
+    <nav class="heat2" id="nav-container">
+        <article class="heat2 header-item">
```
Comment (samgaco) [inline index.html:12]: Implemented as it was required :+1:

**Review by samgaco — APPROVED** [review]
#### Hi @fedgut and @jalasem :high_brightness: 

#### The project is approved :100: Congratulations :clap:  You can now merge to the master branch and submit the project completion form.

---------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**